aries-site/src/examples/components/skeleton/SkeletonAnatomy.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Box, Button, Card, CardBody, Paragraph } from 'grommet';
+
+export const SkeletonAnatomy = ({ ...rest }) => (
+  <Card id="container" flex={false} width="large" {...rest}>
+    <CardBody align="start" gap="medium">
+      <Box align="center" gap="small" direction="row">
+        <Box
```

Comment:
Why not use Text in a `<Box skeleton />` or at least `<Skeleton>` for these? More likely to just work if theme changes.

Reply:
I wanted the `rounding` and Im not able to pass `round` to the parent `box` of `text`
so I changed to `skeleton`